.github/workflows/scripts/e2e.maven.default.verify.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source "./.github/workflows/scripts/e2e-verify.common.sh"
+
+RUNNER_DEBUG=${RUNNER_DEBUG:-}
+if [[ -n "${RUNNER_DEBUG}" ]]; then
+    set -x
+fi
+
+go env -w GOFLAGS=-mod=mod
+
+verify_provenance_content() {
+    e2e_verify_predicate_subject_name "${ATTESTATION}" "test-java-project-0.1.19.jar"
```

Comment:
nit: use an env variable for the jar name, like https://github.com/slsa-framework/example-package/blob/main/.github/workflows/scripts/e2e.go.default.verify.sh#L50. You'll need to set the env variable in the workflow like https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.tag.main.config-ldflags-assets-draft-tag.slsa3.yml#L105-L108

This allows more flexibility